tests/unit/s2n_resume_test.c

```diff
@@ -196,6 +210,22 @@ int main(int argc, char **argv)
             TICKET_AGE_ADD_BYTES,
             SECRET_LEN,
             SECRET,
+            EMPTY_EARLY_DATA_SIZE,
+        };
+
+        uint8_t tls13_ticket_with_early_data[] = {
+            S2N_TLS13_SERIALIZED_FORMAT_VERSION,
+            S2N_TLS13,
+            TLS_AES_128_GCM_SHA256,
+            TICKET_ISSUE_TIME_BYTES,
+            TICKET_AGE_ADD_BYTES,
+            SECRET_LEN,
+            SECRET,
+            0x00, 0x00, 0x00, NONEMPTY_EARLY_DATA_SIZE,
```

Comment:
Early data size is 4 bytes. Rather than have both NONEMPTY_EARLY_DATA_SIZE_BYTES and NONEMPTY_EARLY_DATA_SIZE, I've just added the necessary zero bytes separately.